packages/ses/src/strict-scope-terminator.js

```diff
@@ -21,7 +27,7 @@ const { Fail, quote: q } = assert;
  * create one and share it between all Proxy handlers.
  */
 export const alwaysThrowHandler = new Proxy(
-  immutableObject,
+  onlyFrozenObject,
   freeze({
     get(_shadow, prop) {
       Fail`Please report unexpected scope handler trap: ${q(String(prop))}`;
```

Comment:
So thinking about this, unless we do something wrong inside ses itself, we should never trigger this for the `isNonTrapping` trap, right? And even if we did, it would be a safe failure.

Reply:
Seems so. So no suggest change, right? Resolving.